doc/builders/trivial-builders.chapter.md

```diff
@@ -0,0 +1,52 @@
+# Trivial builders {#chap-trivial-builders}
+
+Nixpkgs provides a couple of functions that help with building derivations. The most important one, `stdenv.mkDerivation`, has already been documented above. The following functions wrap `stdenv.mkDerivation`, making it easier to use in certain cases.
+
+## `runCommand`{#trivial-builder-runCommand}
```

Comment:
Not sure but I would expect a space between the heading text and attributes to be necessary.

Reply:
Done.